test/gtest/ucs/test_log.cc

```diff
@@ -12,7 +12,7 @@ class log_test : public ucs::test {
 
 public:
     virtual void init() {
-        char ucs_log_spec[64];
+        char ucs_log_spec[70];
```

Comment:
why is this needed?

Reply:
this is fix for GCC compilation warning:
```
ucs/test_log.cc:14:18: error: ‘%s’ directive output may be truncated writing up to 63 bytes into a region of size 59 [-Werror=format-truncation=]
     virtual void init() {
                  ^~~~
ucs/test_log.cc:29:17: note: ‘snprintf’ output between 6 and 69 bytes into a destination of size 64
         snprintf(ucs_log_spec, sizeof(ucs_log_spec), "file:%s", logfile);
         ~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```